src/compiler/core.ts

```diff
@@ -434,7 +434,7 @@ export function flatMapToMutable<T, U>(array: readonly T[] | undefined, mapfn: (
 }
 
 /** @internal */
-export function* flatMapIterator<T, U>(iter: Iterable<T>, mapfn: (x: T) => readonly U[] | Iterable<U> | undefined) {
+export function* flatMapIterator<T, U>(iter: Iterable<T>, mapfn: (x: T) => readonly U[] | Iterable<U> | undefined): Generator<U, void, any> {

```

Comment:
@rbuckton 
This is a case where I changed it to `any` to allow `TNext` of `Iterable<U>` to accept values

Reply:
I think typing this as `undefined` will work for both 5.5 and 5.6:
```suggestion
export function* flatMapIterator<T, U>(iter: Iterable<T>, mapfn: (x: T) => readonly U[] | Iterable<U> | undefined): Generator<U, void, undefined> {
```

Reply:
This did not build for me 😢

Reply:
Then I suppose `any` is fine for now.